src/containers/Confirm/Identity/index.tsx

```diff
@@ -159,9 +160,8 @@ class IdentityComponent extends React.Component<Props, IdentityState> {
         const onSelectNationality = value => this.selectNationality(dataNationalities[value]);
 
         /* tslint:disable */
-        countries.registerLocale(require("i18n-iso-countries/langs/en.json"));
-        countries.registerLocale(require("i18n-iso-countries/langs/ru.json"));
-        countries.registerLocale(require("i18n-iso-countries/langs/zh.json"));
+        languages.length > 0 ? languages.map((l: string) => countries.registerLocale(require(`i18n-iso-countries/langs/${l}.json`))) : countries.registerLocale(require('i18n-iso-countries/langs/en.json'));
```

Comment:
an then you can just map through langs array without ` : countries.registerLocale(require('i18n-iso-countries/langs/en.json'));`